lib/src/main/scala/spinal/lib/CrossClock.scala

```diff
@@ -130,10 +135,12 @@ object ResetCtrl{
     )
 
     val solvedOutputPolarity = if(outputPolarity == null) clockDomain.config.resetActiveLevel else outputPolarity
+    val falsePathAttrs = List(crossClockFalsePath, new crossClockFalsePathSource(input, destIsReset = true))
     samplerCD(BufferCC(
       input       = (if(solvedOutputPolarity == HIGH) False else True) ^ inputSync,
-      init        = (if(solvedOutputPolarity == HIGH) True  else False),
-      bufferDepth = bufferDepth)
+      init        = if(solvedOutputPolarity == HIGH) True  else False,
+      bufferDepth = bufferDepth,
+      allBufAttributes = falsePathAttrs)
```

Comment:
we normally have max_delay constraints as well on these, but I could not find any official docs on that (and I guess the async_reg attribute should take care of that anyways, at least for the xilinx case?)

only think that I could find was: https://support.xilinx.com/s/question/0D52E00006hpdDoSAI/ucf-constraints-for-reset-synchronizer?language=en_US - hardly authoritative